auth_mellon_cookie.c

```diff
@@ -78,12 +81,18 @@ static const char *am_cookie_params(request_rec *r)
     secure_cookie = cfg->secure;
     http_only_cookie = cfg->http_only;
 
+    if ( cfg->cookie_expires > -1 ) {
+      apr_rfc822_date(rbuf, apr_time_now() + (APR_USEC_PER_SEC * cfg->cookie_expires));
+      snprintf(cookie_expires, sizeof(cookie_expires)-1, "; Expires=%s", rbuf);
```

Comment:
Instead of using the somewhat hairy `snprintf` function, I think it is better to use `apr_psprintf` here; it will simplify the code and avoid `snprintf`, which is always a good thing.